scout/server/blueprints/cases/views.py

```diff
@@ -680,6 +680,9 @@ def delivery_report(institute_id, case_name):
     else:
         delivery_report = case_obj["delivery_report"]
 
+    out_dir = os.path.abspath(os.path.dirname(delivery_report))
```

Comment:
🤷

Reply:
yeah, that fixes the demo delivery report not found